src/game_engine/Space_Game.java

```diff
@@ -89,7 +90,7 @@ public void runGame() {
 	}
 	
 	public void detectProjectileCollision() {
-		double MIN_EQUAL_DIFF = 0.0001;

```

Comment:
We need to take the blocks and compare each block to see if there is a collision detection